data/saas/config/mandrill_config.yml

```diff
@@ -0,0 +1,44 @@
+saas_config:
+  fides_key: <instance_fides_key>
+  name: Mandril SaaS Config
+  type: mandrill
+  description: A sample schema representing the Mandril connector for Fided
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: api_key
+
+  client_config:
+    protocol: https
+    host: <domain>
+
+  consent_requests:
+    opt_in:
+      - method: POST
+        path: /allowlists/add
+        body: |
+          {
+            "key": "<api_key>",
+            "email": "<email>"
+          }
+        data_uses: [advertising.first_party]
```

Comment:
advertising.first_party = 'Email marketing'?

Do we have to worry about customers using different consent preferences in the `config.json` that differ from data uses we spell out in the saas config (or elsewhere)?

Reply:
I think I'd prefer to avoid encoding any `fideslang` taxonomy types in the SaaS config, so it can remain purely, un-opinionated HTTP details.

Instead, let's think about how we'd customize the connector itself with whether it should or shouldn't be executed. The most primitive version of this, for example, would be a config setting on the connector config.

Reply:
That makes sense @NevilleS thanks for making this distinction.